angular/core/services/intercom_service.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('loomioApp').factory 'IntercomService', ($rootScope, $window, App
     key: group.key
     name: group.name
     description: group.description.substring(0, 250)
-    admin_link: AppConfig.baseUrl+"/admin/groups/#{group.key}"
+    admin_link: AppConfig.baseUrl+"admin/groups/#{group.key}"
```

Comment:
I think my preferred syntax would be

```
LmoUrlService.route(model: @group, absolute: true, namespace: 'admin')
```

Looks like we handle that case already in lmo_url_service on line 13:
```
  "/".concat(route).replace('//', '/')
```